README.md

```diff
@@ -140,6 +140,43 @@ assessment, err := client.RegisterLogin(&incognia.Login{
 })
 ```
 
+### Registering Payment or Login without evaluating its risk assessment
+
+Turning off the risk assessment evaluation allows you to register a new transaction (Login or Payment), but the response will be an empty response (`TransactionAssessment` and `SignupAssessment`). For instance, if you're using the risk assessment only for some payment transactions you still should register all the other ones: this will avoid any bias on the risk assessment computation.
```

Comment:
```suggestion
Turning off the risk assessment evaluation allows you to register a new transaction (Login or Payment), but the response (`TransactionAssessment` and `SignupAssessment`) will be empty. For instance, if you're using the risk assessment only for some payment transactions, you should still register all the other ones: this will avoid any bias on the risk assessment computation.
```